src/cow-react/modules/limitOrders/containers/OrdersWidget/hooks/useLimitOrdersList.ts

```diff
@@ -1,11 +1,30 @@
 import { useWeb3React } from '@web3-react/core'
 import { useOrders } from 'state/orders/hooks'
-import { useMemo } from 'react'
+import { useCallback, useMemo } from 'react'
 import { Order, OrderStatus } from 'state/orders/actions'
+import BigNumber from 'bignumber.js'
+import { getOrderFilledAmount } from '@cow/modules/limitOrders/utils/getOrderFilledAmount'
+import { getOrderSurplus } from '@cow/modules/limitOrders/utils/getOrderSurplus'
+import { getOrderExecutedAmounts } from '@cow/modules/limitOrders/utils/getOrderExecutedAmounts'
+import JSBI from 'jsbi'
 
 export interface LimitOrdersList {
-  pending: Order[]
-  history: Order[]
+  pending: ParsedOrder[]
+  history: ParsedOrder[]
+}
+
+export interface ParsedOrder extends Order {
+  executedBuyAmount?: JSBI
```

Comment:
Why all parameters are optional?

Reply:
Mainly because apiAdditionalInfo is optional on order

Reply:
But in `useLimitOrdersList()` I see that you always set all parameters except of:
```
surplusFee?: string
partiallyFilled?: boolean
fullyFilled?: boolean
```